flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/source/reader/SourceReaderBaseTest.java

```diff
@@ -143,7 +143,11 @@ public void testMultipleSplitsWithDifferentFinishingMoments() throws Exception {
                         .setBlockingFetch(false)
                         .build();
         MockSourceReader reader =
-                new MockSourceReader(elementsQueue, () -> mockSplitReader, getConfig(), null);
+                new MockSourceReader(
+                        elementsQueue,
+                        () -> mockSplitReader,
+                        getConfig(),
+                        new TestingReaderContext());
```

Comment:
Why did you change it from `null` to a value?

Reply:
Because the `SourceReaderBase` is now accessing the metric group in ctor.